test/e2e/manifests/internalserviceexport-crd.yaml

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
Hi Arvind! Is this linking back to the file in `v1alpha1`? I feel that it's a bad pattern.